DevCycle.SDK.Server.Common/Model/Local/DVCLocalOptions.cs

```diff
@@ -36,9 +42,10 @@ public class DVCLocalOptions : IDVCOptions
             CdnCustomHeaders = cdnCustomHeaders;
             EventsApiCustomHeaders = eventsApiCustomHeaders;
             DisableAutomaticEvents = disableAutomaticEvents;
-            DisableCustomEvents = DisableCustomEvents;
+            DisableCustomEvents = disableCustomEvents;
             if (maxEventsInQueue > 1000) maxEventsInQueue = 1000;
             MaxEventsInQueue = maxEventsInQueue;
+            FlushEventQueueSize = flushEventQueueSize;
```

Comment:
Also need to make sure that all these options have the same validation applied: https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/eventQueue.ts#L48

Reply:
Yeah, the flush interval itself is not implemented yet. I'll make sure to hit all of these when that part is built.

Reply:
Going to handle validation in a separate ticket